README.md

```diff
@@ -11,6 +11,7 @@ To deploy the website on a machine running docker run the following commands:
 ```bash
 git clone https://github.com/Imageomics/Andromeda.git
 cd Andromeda/
+cp example.env .env
```

Comment:
export BACKEND_WORKERS=$((2*$(nproc)+1))

Reply:
I would rather have the value set within `.env`. Otherwise this step might get lost when someone deploys new changes.

Reply:
I'm going to go ahead and merge, but feel free to open an issue if you want to discuss this further.